itkwidgets/_type_aliases.py

```diff
@@ -16,6 +16,8 @@
 
 Image = Union[np.ndarray, itkwasm.Image, zarr.Group]
 PointSet = Union[np.ndarray, itkwasm.PointSet, zarr.Group]
+CroppingPlanes = List[Dict[str, List[Union[float, int]]]]
```

Comment:
This is a nice addition.

We could make the types more self-documenting by using string `Literal`'s like in:

https://github.com/spatial-image/spatial-image/blob/main/spatial_image.py#L23

Reply:
Awesome!